library/centos

```diff
@@ -2,7 +2,16 @@ Maintainers: The CentOS Project <cloud-ops@centos.org> (@CentOS)
 GitRepo: https://github.com/CentOS/sig-cloud-instance-images.git
 Directory: docker
 
-Tags: latest, centos8, 8, centos8.1.1911, 8.1.1911
+Tags: latest, centos8, 8, centos8.2.2004, 8.2.2004
+GitFetch: refs/heads/CentOS-8-x86_64
+GitCommit: 5125788a241618b4d37254050d0bddcbd5b7df33
+arm64v8-GitFetch: refs/heads/CentOS-8-aarch64
+arm64v8-GitCommit: e95dba14f002e894a02ac0e7b8d51cb82035a30e
+ppc64le-GitFetch: refs/heads/CentOS-8-ppc64le
+ppc64le-GitCommit: 006350a458663e8b30ac7fcf1f73f60933067c7c
+Architectures: amd64, arm64v8, ppc64le
+
+Tags: centos8.1.1911, 8.1.1911
```

Comment:
I'd love to revisit the conversation around these older (unsupported) snapshot releases -- removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)